purchase_invoice_payment_term/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Inherit in invoice the payment terms of the purchase order.
```

Comment:
Improve a little bit this description, indicating previous (standard) behavior and achieved one when installing this addon